pandas/tests/io/json/test_pandas.py

```diff
@@ -1601,3 +1601,16 @@ def test_json_indent_all_orients(self, orient, expected):
     def test_json_negative_indent_raises(self):
         with pytest.raises(ValueError, match="must be a nonnegative integer"):
             pd.DataFrame().to_json(indent=-1)
+
+    @pytest.mark.parametrize(
+        "json,expected",
+        [
+            (
+                json.dumps([{"col": "31900441201190696999"}, {"col": "3190044"}]),
+                DataFrame({"col": [31900441201190696999.0, 3190044.0]}),
+            ),
+        ],
+    )
+    def test_frame_int_overflow(self, json, expected):
+        result = read_json(json)
```

Comment:
Can you add the GH issue number as a comment here?

Reply:
Sure, Issue no. #30320

Reply:
Can you add this to the code? Should see examples on other tests